contracts/examples/Template.sol

```diff
@@ -85,7 +85,7 @@ contract Template is TemplateBase {
         // Initialize apps
         vault.initialize();
         tokenManager.initialize(token, true, 0);
-        redemptions.initialize(vault, tokenManager);
+        redemptions.initialize(vault, tokenManager, new address[](0));
```

Comment:
i think that we need to add the redeemable token RDT to that list right @fabriziovigevani ?

Reply:
No, the RDT token would be the `tokenManager`. 
The list is for redeemable tokens (eth, dai, etc)